build.gradle

```diff
@@ -94,6 +94,10 @@ dependencies {
     }
     compile "commons-io:commons-io:2.10.0"
 
+    testImplementation platform("org.junit:junit-bom:5.7.2")
+    testImplementation "org.junit.jupiter:junit-jupiter"
+    testRuntimeOnly "org.junit.vintage:junit-vintage-engine"
+    testRuntimeOnly "org.junit.platform:junit-platform-launcher"
     testCompile "junit:junit:4.13"
```

Comment:
What's the effect of keeping this here?

Reply:
Hmm, so I thought that without this, JUnit4 would fail to run, but as it's brought in as a transitive dependency of `junit-vintage-engine`, it's not required - I'll amend :+1: